src/func.cc

```diff
@@ -656,6 +657,28 @@ void ShellFuncNoRerun(const std::vector<Value*>& args,
   ShellStatusVar::SetValue(returnCode);
 }
 
+void VarVisibilityFunc(const std::vector<Value*>& args,
+                       Evaluator* ev,
+                       std::string* s) {
+  std::string arg = args[0]->Eval(ev);
+  std::vector<std::string> prefixes;
+
+  std::stringstream ss(args[1]->Eval(ev));
+  std::string prefix;
+  while (ss >> prefix) {
+    prefixes.push_back(prefix);
```

Comment:
There should be some checks here:
- The path doesn't start with a / (isn't absolute)
- The path is equal to `NormalizePath(path)`
- One path isn't a prefix of another